CoreLibrary/src/CoreLib/Constants.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Constants.h
+ *
+ *  Created on: Aug 25, 2015
+ *      Author: core
+ */
+
+
+#ifndef SRC_CORELIB_CONSTANTS_H_
+#define SRC_CORELIB_CONSTANTS_H_
+
+#include <string>
+
+enum motors{
+
+};
+
+
+enum controllerInputs{
+
+};
+
+enum digitalSensors{

```

Comment:
should probably be DigitalSensor—types generally start with an uppercase letter to distinguish them from values, and are singular nouns.

(Same with the other types in this file)